libsoftwarecontainer/src/gateway/iptableentry.h

```diff
@@ -67,6 +67,18 @@ class IPTableEntry
      */
     ReturnCode applyRules();
 
+    /**
+     * @brief Interprets a rule to iptables applicable string
+     * @return string indicating interpreted rule
+     */
+    std::string interpretRule(Rule rule);
+
+    /**
+     * @brief Interprets defaultTarget rule to iptables applicable string
+     * @return string indicating interpreted policy
```

Comment:
Actually this check is not supposed to be interfered on real scenarios. 
Normally NetworkGateway parser eliminates the possibility. 
But in case for maybe future use of it in another gateway or class it should be beneficial.